extras/iris_pipeline_project/README.md

```diff
@@ -0,0 +1,182 @@
+# OVMS Iris Pipeline Example
+
+This repository demonstrates how to use OpenVINO Model Server (OVMS) with a custom Mediapipe pipeline for the Iris dataset, including both model training and inference through a Python client.
+At the moment, it supports Logistic regression and KMeans.
+
+---
+
+## Step 1: Clone the Repository
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/extras/iris_pipeline_project
+```
+---
+
+
+## Step 2: Build and Run OVMS Docker Image
+
+### 2.1. Build the Docker Image
+
+```bash
+docker build --no-cache -t prototype_iris .
+```
+
+### 2.2. Run the OVMS Container
+
+```bash
+ docker run --rm -it -v $(pwd):/workspace -p 9000:9000 prototype_iris --config_path /workspace/model_config.json --port 9000 --log_level DEBUG
+```
+- **Note:** Adjust `$(pwd)` if you are running from a different working directory.
+
+---
+
+##  Step 3: Project Structure
+
+```
+client/
+  ├── client_inference.py
+  └── client_train.py
+data_folder/
+  ├── iris_train.csv
+  └── iris_test.csv
+pipeline/
+  ├── __pycache__/
+  ├── graph.pbtxt
+  ├── model.py
+  └── ovmsmodel.py
+Dockerfile
+model_config.json
+kmeans_params.json
+hyperparams.json
+
+```
+
+---
+
+## Step 4: Run Training and Inference
+
+### 4.1. Training
+
+```bash
+python client/client_train.py train iris_train.csv Species --params hyperparams.json --encode Species --model_class LogisticRegressionTorch
+
+python client/client_train.py train iris_train.csv Species --params kmeans_params.json --encode Species --model_class KMeansSkLearn
+
+
+```
+
+### 4.2. Inference
+
+```bash
+python client/client_inference.py infer data_folder/iris_test.csv --target_column Species  --model_class LogisticRegressionTorch
+
+python client/client_inference.py infer iris_train_nolabel.csv  --target_column Species --model_class KMeansSkLearn
+
+```
+
+---
+
+For Enabling accelerator support:
+Manually set the ```bool - (use_ipex/use_oneDAL)``` in model.py file under "pipeline" directory to either True/False depending on the necessity.
```

Comment:
We needed that for benchmarks.In the guide we should not require user to modify the code. 
Either remove it completely and use acceleration by default or pass it in hyperparameters JSON.